### PR TITLE
Bug in Router.php match function

### PR DESCRIPTION
**Pull request by bpiwowar**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #3094
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -125,7 +125,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.5.x-dev"
```
Comment (dunglas) [inline composer.json]: This must be reverted

File: src/Bridge/Symfony/Routing/Router.php [hunk 2]
```diff
@@ -71,7 +71,11 @@ public function getRouteCollection()
     public function match($pathInfo)
     {
         $baseContext = $this->router->getContext();
-        $pathInfo = str_replace($baseContext->getBaseUrl(), '', $pathInfo);
+        $baseURL = $baseContext->getBaseUrl();
+        $p = strpos($pathInfo, $baseURL);
```
Comment (dunglas) [inline src/Bridge/Symfony/Routing/Router.php]: `$p` is unused, this line should be removed

Comment (bpiwowar) [reply src/Bridge/Symfony/Routing/Router.php]: Done

File: composer.json (hunk 1 above)
Comment (bpiwowar) [reply composer.json]: Done - I had modified the current master, but switched to the 2.5 branch

Comment (bpiwowar) [reply composer.json]: done

File: src/Bridge/Symfony/Routing/Router.php (hunk 2 above)
Comment (bpiwowar) [reply src/Bridge/Symfony/Routing/Router.php]: done

**Comment by alanpoulain** [comment]
Superseded by https://github.com/api-platform/core/pull/3776.

**Pull request closed without merging**